packages/examples/iotcc_sensor_tag.py

```diff
@@ -0,0 +1,265 @@
+    # Accelerometer x,y,z in g
+    x, y, z = sensor_tag_collector.get_accelerometer()
+    # Magnitude of acceleration
+    # ∣a⃗∣=√ (x*x+y*y+z*z)
+    vib = int(math.sqrt((x*x + y*y + z*z)))
+    return vib
+
+
+def get_rpm(sensor_tag_collector):
+    # RPM of Z-axis
+    # Average of 5 samples
+    _rpm_list = []
+    while True:
+        if len(_rpm_list) == 5:
+            rpm = 0
+            for _ in _rpm_list:
+                rpm += _
+            rpm = int(rpm / 5)
+            break
+        else:
+            z_degree = sensor_tag_collector.get_gyroscope()[2]
+            # (°/s to RPM)
+            _rpm_list.append(int((abs(z_degree) * 0.16667)))
+    return rpm
+
+
+class PackageClass(LiotaPackage):
+    def run(self, registry):
+        import copy
+        from liota.entities.metrics.metric import Metric
+        from liota.entities.devices.sensor_tag import Sensors, SensorTagCollector
+
+        # Acquire resources from registry
+        iotcc_edge_system = copy.copy(registry.get("iotcc_edge_system"))
+        iotcc = registry.get("iotcc")
+
+        # Get values from configuration file
+        config_path = registry.get("package_conf")
+        config = {}
+        execfile(config_path + '/sampleProp.conf', config)
+
+        # Create metrics
+        self.metrics = []
+        # Operational metrics of EdgeSystem
+        cpu_utilization_metric = Metric(
+            name="CPU Utilization",
+            unit=None,
+            interval=10,
+            aggregation_size=2,
+            sampling_function=read_cpu_utilization
+        )
+        reg_cpu_utilization_metric = iotcc.register(cpu_utilization_metric)
+        iotcc.create_relationship(iotcc_edge_system, reg_cpu_utilization_metric)
+        # call to start collecting values from the device or system and sending to the data center component
+        reg_cpu_utilization_metric.start_collecting()
+        self.metrics.append(reg_cpu_utilization_metric)
+
+        cpu_procs_metric = Metric(
+            name="CPU Process",
+            unit=None,
+            interval=6,
+            aggregation_size=8,
+            sampling_function=read_cpu_procs
+        )
+        reg_cpu_procs_metric = iotcc.register(cpu_procs_metric)
+        iotcc.create_relationship(iotcc_edge_system, reg_cpu_procs_metric)
+        reg_cpu_procs_metric.start_collecting()
+        self.metrics.append(reg_cpu_procs_metric)
+
+        mem_free_metric = Metric(
+            name="Memory Free",
+            unit=None,
+            interval=10,
+            sampling_function=read_mem_free
+        )
+        reg_mem_free_metric = iotcc.register(mem_free_metric)
+        iotcc.create_relationship(iotcc_edge_system, reg_mem_free_metric)
+        reg_mem_free_metric.start_collecting()
+        self.metrics.append(reg_mem_free_metric)
+
+        # Connects to the SensorTag device over BLE
+        self.sensor_tag_collector = SensorTagCollector(device_name=config['DeviceName'], device_mac=config['DeviceMac'],
```

Comment:
Create a separate device package and metrics package. Initialization of device shouldn't be part of this device metrics package.